packages/core/src/PartialExecutionUtils/removeDirtyNodes.ts

```diff
@@ -0,0 +1,12 @@
+import type { IRunData } from 'n8n-workflow';
+
+export function removeDirtyNodes(runData: IRunData, dirtyNodeNames: string[]) {
```

Comment:
can we use `omit(runData, ...dirtyNodeNames)` instead of this additional function?